utils/data-ingestor.js

```diff
@@ -123,19 +123,26 @@ class DataIngestor {
     const specialType = updatedRow.get('specialType');
     if (specialType === undefined) return updatedRow;
 
+    // unset percent and percent_m
+    updatedRow = this.unsetPercents(updatedRow);
+
+    // do not recalculate values for special rows of type 'noPercentOnly'
+    if (specialType === 'noPercentOnly') return updatedRow;
+
     // recalculate sum, and optionally m & cv
     const variable = updatedRow.get('variable');
     try {
       const year = this.isPrevious ? PREV_YEAR : CUR_YEAR;
       const { options } = find(specialCalculationConfigs[this.profileName], ['variable', variable]);
 
-      updatedRow = this.recomputeSum(updatedRow, specialType, variable, year, options);
+      let wasCoded;
+      ({ updatedRow, wasCoded } = this.recomputeSum(updatedRow, specialType, variable, year, options));
```

Comment:
Whoa! I forgot you had to do it this way... so "destructuring" stuff _as reassignments_ requires `()`.

Reply:
yeah :) twas funky but i luv to learn a funky javascripty thing